index.html

```diff
@@ -68,219 +63,246 @@ <h1>
-        <h1 class="myself-header">about myself</h1>
-        <p class="myself-paragraph">
-          Hello I’m a software developer! I can help you build a product ,
-          feature or website Look through some of my work and experience! If you
-          like what you see and have a project you need coded, don’t hestiate to
-          contact me.
-        </p>
-        <a class="link-up" href="#">LET'S CONNECT</a>
-
-        <ul class="social-media-icons">
-          <li>
-            <a href="https://github.com/Goodluck5956"
-              ><i class="fab fa-github"></i
-            ></a>
-          </li>
-          <li>
-            <a href="www.linkedin.com/in/lucky-james-osuagwu-a8677b171"
-              ><i class="fab fa-linkedin-in"></i
-            ></a>
-          </li>
-          <li>
-            <a href=""><i class="fab fa-angellist"></i></a>
-          </li>
-          <li>
-            <a href=""><i class="fab fa-twitter"></i></a>
-          </li>
-          <li>
-            <a href=""><i class="fab fa-medium-m"></i></a>
-          </li>
-        </ul>
-
-        <button class="btn-body" type="submit" name="submit" id="submit">
-          get my resume
-        </button>
-        <ul class="language">
-          <li class="downlinks">languages</li>
-          <li><img src="./templates/Union1.png" alt="dropdown-icon" /></li>
-        </ul>
-        <nav class="program-languages">
-          <ul class="script">
-            <li><img src="./templates/Ellipse5.svg" alt="js-logo" /></li>
-            <li>JavaScript</li>
+        <div class="flex-item1 flex-item">
+          <h1 class="myself-header">
+            About<br />
+            Myself
+          </h1>
+          <p class="myself-paragraph">
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </p>
+          <a class="connect" href="#">LET'S CONNECT</a>
+          <nav class="icon-body">
+            <ul class="social-icons">
+              <li>
+                <a href="https://github.com/Goodluck5956"
+                  ><i class="fab fa-github"></i
+                ></a>
+              </li>
+              <li>
+                <a href="www.linkedin.com/in/lucky-james-osuagwu-a8677b171"
+                  ><i class="fab fa-linkedin-in"></i
+                ></a>
+              </li>
+              <li>
+                <a href=""><i class="fab fa-angellist"></i></a>
+              </li>
+              <li>
+                <a href=""><i class="fab fa-twitter"></i></a>
+              </li>
+              <li>
+                <a href=""><i class="fab fa-medium-m"></i></a>
+              </li>
+            </ul>
+          </nav>
+          <form action="#">
+            <button class="btn-body" type="submit" name="submit" id="submit">
```

Comment:
- [ ] Like all the other buttons, add the button decorations like hover, active, and press